templates/types/extractor/fastapi/app/services/extractor.py

```diff
@@ -1,18 +1,34 @@
-from typing import Optional
-
+import logging
 from app.engine import get_query_engine
-from app.models.output import Output
+from app.services.model import IMPORTS
+
+logger = logging.getLogger("uvicorn")
+
+
+class InvalidModelCode(Exception):
+    pass
 
 
 class ExtractorService:
     @staticmethod
-    def parse_json_schema(json_schema: Optional[str] = None):
-        # TODO: Init model from json schema once llama_index supports Pydantic V2
-        return Output
+    def _parse_code(model_code: str):
+        try:
+            python_code = f"{IMPORTS}\n\n{model_code}"
+            logger.debug(python_code)
+            namespace = {}
+            exec(python_code, namespace)
+            # using the last object that the user defined in `model_code` as pydantic class
```

Comment:
Then i think can have add notice in the UI to tell user remember to specify it at the end. Or we can of course preset a variable like
```
<user model code here>

// Set your model name here
use_model=ModelName
```